components/common/WalletModal.vue

```diff
@@ -108,6 +106,7 @@ export default class WalletModal extends Vue {
   set account(account: string) {
     this.$emit('close')
     this.$store.dispatch('setAuth', { address: account })
+    localStorage.setItem('selected_account', account)
```

Comment:
Something better than snek case? `selected_account`

like: `kodauth`, `auth`, `selectedAccount`

Reply:
Also I would move it to the constants or something :) 
One typo aaand It's gone

Reply:
I've already had problems with camelCase key in localStorage, let me know wich key you want to use